src/modules/Providers/modals/ModalHOC/ModalContext.tsx

```diff
@@ -0,0 +1,9 @@
+import { createContext, type ReactNode } from 'react';
+
+export type ModalContextType = {
+  showModal: (modal: ReactNode) => void;
+  toggleModal: () => void;
+};
+
+// Creating the context.
+export const ModalContext = createContext<ModalContextType | undefined>(undefined);
```

Comment:
I think console provides us with the useModal hook from the SDK, which adds context to the entire app. Can you please check?